components/typography/TextLink.tsx

```diff
@@ -0,0 +1,34 @@
+import { twMerge } from 'tailwind-merge';
+
+interface TextLinkProps {
+  href: string;
+  className?: string;
+  target?: string;
+  children?: React.ReactNode;
+  id?: string;
+}
+
+const TextLink: React.FC<TextLinkProps> = ({
+  href,
+  className,
+  target,
+  children,
+  id
+}) => {
+  const classNames = twMerge(`text-secondary-500 underline hover:text-gray-800 font-medium transition ease-in-out duration-300 ${className || ''}`);
+
+  return (
+    <a 
```

Comment:
We should use `link` tag also here as previously used in js file

Reply:
i tried adding that but shows some kind of an error

Reply:
So, the main problem is, replace `a` with `Link` tag here and add the same CSS properties inside `span` as child element.